### DISCUSSION TITLE
No priceV2 on variants

### DISCUSSION THREAD

**Discussion by chrisvince** [gatsbyjs/gatsby#32088] [Ideas / Feature Requests]
Hey guys!

It seems that there is no `priceV2` data point on variants, only `price` (which doesn't include currencyCode). It looks like there is `priceRangeV2` on products, but not `priceV2` on variants.
<img width="1009" alt="Screen Shot 2021-06-19 at 14 46 45" src="https://user-images.githubusercontent.com/13654976/122652572-85881480-d10d-11eb-9289-a3e0b479145b.png">

**Comment by sslotsky** [comment]
We pull data from the admin/bulk API which does not expose a `priceV2` for variants in the current version.

https://shopify.dev/docs/admin-api/graphql/reference/products-and-collections/productvariant#fields-2021-04

I admit that I don't understand what function `priceV2` serves so it's hard for me to say whether or not the data we have supports your use case.

**Comment by chrisvince** [comment]
Ah I didn't realize priceV2 isn't part of the admin API. It looks like instead `presentmentPrices` should be used instead which lists all the prices in all currencies (`price` only gives the price in the **default** currency).

It doesn't look like `presentmentPrices` is available either, is that an easy add?

Documentation:
https://shopify.dev/docs/admin-api/graphql/reference/products-and-collections/productvariant#presentmentprices-2021-04

**Comment by sslotsky** [comment]
It's been requested in another open issue. We plan to add it but there's a blocking refactor.

**Comment by chrisvince** [comment]
Amazing! Thanks @sslotsky!
